<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Books</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>741</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Entry No.</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Date Posted</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDateEdit" name="dateEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Date Event</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDateEdit" name="dateEdit_2"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Account</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Debit</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Credit</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEdit_3"/>
     </item>
     <item row="6" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox">
       <property name="maximum">
        <double>999999.989999999990687</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
       <property name="maximum">
        <double>999999.989999999990687</double>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QComboBox" name="comboBox_2"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="lineEdit_4"/>
     </item>
     <item row="7" column="3">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
       <property name="maximum">
        <double>999999.989999999990687</double>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QComboBox" name="comboBox_3"/>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="lineEdit_5"/>
     </item>
     <item row="7" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_5">
       <property name="maximum">
        <double>999999.989999999990687</double>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_6">
       <property name="maximum">
        <double>999999.989999999990687</double>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_7">
       <property name="maximum">
        <double>999999.989999999990687</double>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QComboBox" name="comboBox_4"/>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="lineEdit_6"/>
     </item>
     <item row="9" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_8">
       <property name="maximum">
        <double>999999.989999999990687</double>
       </property>
      </widget>
     </item>
     <item row="9" column="3">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_9">
       <property name="maximum">
        <double>999999.989999999990687</double>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QComboBox" name="comboBox_5"/>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="lineEdit_7"/>
     </item>
     <item row="10" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_10">
       <property name="maximum">
        <double>999999.989999999990687</double>
       </property>
      </widget>
     </item>
     <item row="10" column="3">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_11">
       <property name="maximum">
        <double>999999.989999999990687</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="2">
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Copy to Accounts</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>460</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Erase Form</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>460</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>762</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Books"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionView_GJ_Entry"/>
    <addaction name="actionView_GL_Account"/>
    <addaction name="actionView_GJ"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionView_GJ_Entry"/>
   <addaction name="actionView_GL_Account"/>
   <addaction name="actionView_GJ"/>
   <addaction name="actionAbout_Books"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open Connection</string>
   </property>
  </action>
  <action name="actionAbout_Books">
   <property name="text">
    <string>About Books</string>
   </property>
  </action>
  <action name="actionView_GJ_Entry">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>View GJ Entry</string>
   </property>
  </action>
  <action name="actionDelete_GJ_Entry">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Delete GJ Entry</string>
   </property>
  </action>
  <action name="actionView_GL_Account">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>View GL Account</string>
   </property>
  </action>
  <action name="actionView_GJ">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>View GJ</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
